.github/workflows/upgrade-testing.yml

```diff
@@ -35,19 +35,25 @@ jobs:
   # Spawns upgrade testing from WordPress 6.x versions with MySQL.
   upgrade-tests-wp-6x-mysql:
     name: ${{ matrix.wp }} to ${{ inputs.new-version && inputs.new-version || 'latest' }}
-    uses: WordPress/wordpress-develop/.github/workflows/reusable-upgrade-testing.yml@trunk
+    uses: desrosj/wordpress-develop/.github/workflows/reusable-upgrade-testing.yml@add/mysql-8.4
```

Comment:
I assume this is just a temporary change until we merge this to the repo? (same question applies to other similar changes)

Reply:
Because we reference the workflow in `trunk`, this is the only way I know of to test changes to our reusable workflow files. This will be skipped in the core commit.